### PR TITLE
feature(ui-ux): add LP token details page

### PR DESCRIPTION
**Pull request by siradji**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:
- To add poolpair token detail screen

#### Which issue(s) does this PR fixes?:

Fixes #

#### Developer Checklist:

- [x] Read your code changes at least once
- [x] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [x] No console errors on web
- [x] Tested on Light mode and Dark mode*
- [x] Your UI implementation visually matched the rendered design*
- [ ] Unit tests*
- [ ] Added e2e tests*
- [x] Added translations*

### CODE REVIEW

File: mobile-app/app/contexts/FavouritePoolpairContext.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  const [favouritePoolpairs, setFavouritePoolpairs] = useState<PoolpairId[]>([])
+
+  useEffect(() => {
+    FavouritePoolpairsPersistence.get().then((poolpair) => {
+      setFavouritePoolpairs(poolpair)
+    }).catch(logger.error)
+  }, [])
+
+  const updateFavouritePoolpairs = async (id: string): Promise<void> => {
+    const _poolpair = favouritePoolpairs.find(poolpair => poolpair.id === id)
```
Comment (fullstackninja864) [inline mobile-app/app/contexts/FavouritePoolpairContext.tsx]: can we convert this to favouritePoolpairs object array to string array?

File: mobile-app/app/screens/AppNavigator/screens/Dex/PoolPairDetailsScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,539 @@
...
+    })
+  }, [navigation])
+
+  if (poolPair === undefined) {
+    return <></>
+  }
+
+  const onLinkPress = async (): Promise<void> => {
+    const url = getTokenUrl(id)
+    await Linking.openURL(url)
```
Comment (fullstackninja864) [inline mobile-app/app/screens/AppNavigator/screens/Dex/PoolPairDetailsScreen.tsx:68]: use `openURL` from `mobile-app/app/api/linking.ts`

File: mobile-app/app/screens/AppNavigator/screens/Dex/PoolPairDetailsScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,539 @@
...
+    <ThemedViewV2
+      style={tailwind('flex flex-row items-center pb-5 mb-5 border-b-0.5')}
+      light={tailwind('border-mono-light-v2-300')}
+      dark={tailwind('border-mono-dark-v2-300')}
+    >
+      <PoolPairIconV2
+        symbolA={props.symbolA}
+        symbolB={props.symbolB}
+      />
+      <View style={tailwind('flex-col ml-3 flex-auto pr-3')}>
```
Comment (fullstackninja864) [inline mobile-app/app/screens/AppNavigator/screens/Dex/PoolPairDetailsScreen.tsx:170]:
```suggestion
      <View style={tailwind('flex-col mx-3 flex-auto')}>
```

File: mobile-app/app/screens/AppNavigator/screens/Dex/PoolPairDetailsScreen.tsx [hunk 4]
```diff
@@ -0,0 +1,539 @@
...
+}
+function PoolPairActionSection ({
+  pair,
+  walletToken,
+  onAdd,
+  onRemove,
+  onSwap
+}: PoolPairActionSectionProps): JSX.Element {
+  return (
+    <View style={tailwind('flex-1 pt-12 ')}>
```
Comment (fullstackninja864) [inline mobile-app/app/screens/AppNavigator/screens/Dex/PoolPairDetailsScreen.tsx:463]:
```suggestion
    <View style={tailwind('flex-1 pt-12')}>
```

File: shared/translations/languages/es.json [hunk 5]
```diff
@@ -1823,5 +1823,18 @@
...
+    "Volume (24H)": "Volume (24H)",
+    "Total liquidity": "Liquidez total",
+    "Pooled {{symbol}}": "{{symbol}} en la piscina",
+    "Your LP tokens": "Your LP tokens",
+    "Tokens in {{symbol}}": "Tokens in {{symbol}}",
+    "APR": "APR",
+    "{{percentage}}% in rewards": "{{percentage}}% in rewards",
+    "{{percentage}}% in commissions": "{{percentage}}% in commissions"
   }
 }
```
Comment (fullstackninja864) [inline shared/translations/languages/es.json:1840]:
```suggestion
}

```

File: shared/translations/languages/it.json [hunk 6]
```diff
@@ -1817,5 +1817,18 @@
...
+    "Volume (24H)": "Volume (24H)",
+    "Total liquidity": "Liquidita' Totale",
+    "Pooled {{symbol}}": "Messo nel pool {{symbol}}",
+    "Your LP tokens": "Your LP tokens",
+    "Tokens in {{symbol}}": "Tokens in {{symbol}}",
+    "APR": "APR",
+    "{{percentage}}% in rewards": "{{percentage}}% in rewards",
+    "{{percentage}}% in commissions": "{{percentage}}% in commissions"
   }
 }
```
Comment (fullstackninja864) [inline shared/translations/languages/it.json:1834]:
```suggestion
}

```

File: shared/translations/languages/zh-Hans.json [hunk 7]
```diff
@@ -1800,5 +1800,18 @@
...
+    "Volume (24H)": "Volume (24H)",
+    "Total liquidity": "总资金值",
+    "Pooled {{symbol}}": "池中总 {{symbol}} 数量",
+    "Your LP tokens": "您的 LP 代币",
+    "Tokens in {{symbol}}": "{{symbol}}中的代币",
+    "APR": "年回报率",
+    "{{percentage}}% in rewards": "{{percentage}}% in rewards",
+    "{{percentage}}% in commissions": "{{percentage}}% in commissions"
   }
 }
```
Comment (fullstackninja864) [inline shared/translations/languages/zh-Hans.json:1817]:
```suggestion
}

```

File: shared/translations/languages/zh-Hant.json [hunk 8]
```diff
@@ -1801,5 +1801,18 @@
...
+    "Volume (24H)": "Volume (24H)",
+    "Total liquidity": "總資金值",
+    "Pooled {{symbol}}": "池中總 {{symbol}} 數量",
+    "Your LP tokens": "您的 LP 代幣",
+    "Tokens in {{symbol}}": "{{symbol}}中的代幣",
+    "APR": "年回報率",
+    "{{percentage}}% in rewards": "{{percentage}}% in rewards",
+    "{{percentage}}% in commissions": "{{percentage}}% in commissions"
   }
 }
```
Comment (fullstackninja864) [inline shared/translations/languages/zh-Hant.json:1818]:
```suggestion
}

```

**Review by fullstackninja864 — CHANGES_REQUESTED** [review]

File: mobile-app/app/screens/AppNavigator/screens/Dex/DexScreen.tsx [hunk 9]
```diff
@@ -107,6 +107,14 @@ export function DexScreen (): JSX.Element {
...
+  const onPress = (id: string): void => {
+    navigation.navigate({
+      name: 'PoolPairDetailsScreen',
+      params: { id: id },
+      merge: true
+    })
+  }
+
   // Search
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
```
Comment (siradji) [inline mobile-app/app/screens/AppNavigator/screens/Dex/DexScreen.tsx:119]: disable-next-line can be removed

Comment (fullstackninja864) [inline mobile-app/app/screens/AppNavigator/screens/Dex/DexScreen.tsx:119]:
```suggestion
```

**Review by fullstackninja864 — DISMISSED** [review]

**Pull request merged**